README.md

```diff
@@ -0,0 +1,166 @@
+| `mergeProps(stateProps, dispatchProps, ownProps)`  | `mergeProps(queryProps, dispatchProps, ownProps)` |
+
+## API Docs
+#### Coming soon!
+
+## Getting Started
+#### Installation
+ A real npm package is coming soon, but to start using right away:
+`npm install https://github.com/lolatravel/realm-react-redux/tarball/v0.0.5`
+
+#### Running the examples
+There's only a basic example right now, more coming soon.
+
+```sh
+git clone git@github.com:lolatravel/realm-react-redux.git
+cd realm-react-redux
+npm install
+cd examples/basic_todo_example
+npm install
+# if you don't have react-native-cli installed globally
+npm install -g react-native-cli
+react-native run-ios
+```
+
+### Basic Example
+```javascript
+// models.js
+export class ToDo {
+    static schema = {
+        name: 'ToDo',
+        primaryKey: 'id',
+        properties: {
+            id: { type: 'string' },
+            name: { type: 'string' },
+            completed: { type: 'bool', default: false }
+        }
+    };
+}
+```
+```javascript
+// actions.js
+export const CREATE_TODO = 'CREATE_TODO';
+export const TOGGLE_TODO = 'TOGGLE_TODO';
+
+export function createTodo(name) {
+    return {
+        type: CREATE_TODO,
+        name
+    };
+}
+
+export function toggleTodo(id) {
+    return {
+        type: TOGGLE_TODO,
+        id
+    };
+}
+```
+```javascript
+// writers.js
+import uuid from 'uuid/v4';
+import { CREATE_TODO, TOGGLE_TODO } from './actions';
+
+export function todoWriter(realm, action) {
+    switch (action.type) {
+        case CREATE_TODO:
+            const { name } = action;
+            realm.create('ToDo', {
+                id: uuid(),
+                name
+            });
+            break;
+
+        case TOGGLE_TODO:
+            const { id } = action;
+            const todos = realm.objects('ToDo').filtered(`id = "${id}"`);
+            if (todos.length === 1) {
+                const todo = todos[0];
+                todo.completed = !todo.completed;
+            }
+            break;
+
+        default:
+            break;
+    }
+}
+```
+```javascript
+// configureStore.js
+import { createRealmStore, combineWriters } from 'realm-react-redux';
+import { todoWriter } from './writers';
+import { ToDo } from './models';
+
+export function configureRealmStore() {
+    // This will create a Realm instance to use in the store, using the options
+    // passed in the second argument. To pass an existing Realm instance instead
+    // you can use createRealmStore(writer, { realm: yourRealmInstance })
+    return createRealmStore(combineWriters([todoWriter]), { schema: [ToDo] });
+}
+```
+```javascript
+// ToDoListContainer.js
+import { bindActionCreators } from 'redux';
+import { realmConnect } from 'realm-react-redux';
+import { createTodo, toggleTodo } from './actions';
+import ToDoList from './ToDoList';
+
+function mapPropsToQueries(realm) {
+    return [realm.objects('ToDo')];
+}
```

Comment:
Basically, except it is `queries.map(q => q.snapshot()` where `queries` is the return value of `mapPropsToQueries` so that it is *almost* immutable. I'm writing full API documentation that goes into more detail on this and just did a simple example here.